### ISSUE TITLE
Ide autostarts

### ISSUE THREAD

**Issue by davidmoshal** [whitecatboard/blockly-environment#16]
noticed that whitecatide starts wcc on port 8080 at startup, with no way to disable this
not so cool.
ubuntu 18.04

D

**Comment by thunderWhitecat** [comment]
@davidmoshal,

Yes, The Whitecat IDE needs a websocket connection on port 8080 to The Whitecat Create Agent. It can be other port, but need this connection, because we can't access the local serial port from a Web App.

What's your use case about the 8080 port? Maybe some conflict with a local service?

**Comment by davidmoshal** [comment]
Understood that Whitecat can't work without a socket, but it's a bit inconsiderate to:  
a) choose the most common default development port (8080), and  
b) set your software to autostart at login 

In my case I had to run lsof to find out what was interfering with by docker and kubernetes development setup, then I had to figure out where that was being set, given that it wasn't a standard Ubuntu startup script (SystemD, Upstart, etc).

Additionally forcing users to use their Google login in order to use you software, that's also a bit much frankly.
